caravel/assets/javascripts/SqlLab/components/QueryHistory.jsx

```diff
@@ -9,12 +9,17 @@ import { Alert } from 'react-bootstrap';
 
 const QueryHistory = (props) => {
   const activeQeId = props.tabHistory[props.tabHistory.length - 1];
-  const queries = props.queries.filter((q) => (q.sqlEditorId === activeQeId));
-  if (queries.length > 0) {
+  const queriesArray = []
+  for (var query_id in props.queries) {
```

Comment:
.filter doesn't work on object, $.map - wouldn't filter the values out. 
Let's keep it as it is - fairly simple construct and we can change it if that gets complicated.